mantidproject/mantid
# Move SNS to use ONCAT for finding files

Since `Facilities.xml` is used automatically by old versions of mantid on startup, change what `SNSDataSearch` points at in the code and remove the old SNS ICAT code.

**To test:**

Search for data from SNS and HFIR with logging at debug level. You should see everything searching ONCAT.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
